code/datums/components/barber.dm

```diff
@@ -86,17 +86,21 @@
 			return 0
 
 	SPAWN_DBG(0)
-		var/new_style = input(user, "Please select style", "Style")  as null|anything in customization_styles + customization_styles_gimmick
+		var/list/region = list("First Hairea" = 1, "Second Hairea" = 2, "Third Hairea" = 3)
```

Comment:
Is "hairea" a word? It it like a portanteau of hair and area? *scream

Reply:
I most certainly hope it isn't a word.

Reply:
Oh good it isnt a word.